Berksfile

```diff
@@ -5,7 +5,7 @@ group :deps do
 end
 
 group :integration do
-  cookbook 'test_windows_ad', path: 'test/fixtures/test_cookbooks/test_windows_ad'
+  cookbook 'test_windows_ad', path: 'test/integration/test_windows_ad'
```

Comment:
This will interfere with kitchen tests. Kitchen assumes that under `test/integration` it will either find a `helpers` directory or test suites. `test_windows_ad` is neither of those.